ui/src/overview.tsx

```diff
@@ -612,6 +678,10 @@ function DemographicCharts({ cohort }: DemographicChartsProps) {
     return value.length > 15 ? value.substr(0, 15).concat("…") : value;
   };
 
+  if (!open) {
```

Comment:
it seems weird to pass this into this component.  shouldn't you just not include this component if your drawer is not open? (as i noted above)